### DISCUSSION TITLE
Reinstall after update not working

### DISCUSSION THREAD

**Discussion by Grasio** [fabianishere/udm-iptv#267] [Q&A]
Last week the software on my Unifi Dream Machine SE was automaticly updated.
This caused the UDM-IPTV script from functioning so I tried to reinstall it like I normally do after a software update.

However, when trying to install the package it gives me the following error:
**W: Repository is broken: udm-iptv:arm64 (= 3.0.2) has no Size information**

root@Dream-Machine-Special-Edition:~# sh -c "$(curl https://raw.githubusercontent.com/fabianishere/udm-iptv/master/install.sh -sSf)"
Downloading packages...
Installing packages...
Hit:1 http://deb.debian.org/debian bullseye InRelease
Hit:2 http://deb.debian.org/debian bullseye-updates InRelease
Hit:3 http://deb.debian.org/debian bullseye-backports InRelease
Hit:4 http://security.debian.org/debian-security bullseye-security InRelease
Hit:5 https://apt.artifacts.ui.com bullseye InRelease
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
The following packages were automatically installed and are no longer required:
  freeswitch freeswitch-mod-amr freeswitch-mod-b64 freeswitch-mod-commands freeswitch-mod-conference freeswitch-mod-console freeswitch-mod-db
  freeswitch-mod-dialplan-asterisk freeswitch-mod-dialplan-xml freeswitch-mod-dptools freeswitch-mod-enum freeswitch-mod-esf
  freeswitch-mod-event-socket freeswitch-mod-expr freeswitch-mod-fifo freeswitch-mod-fsv freeswitch-mod-g723-1 freeswitch-mod-g729
  freeswitch-mod-hash freeswitch-mod-httapi freeswitch-mod-local-stream freeswitch-mod-logfile freeswitch-mod-loopback freeswitch-mod-lua
  freeswitch-mod-native-file freeswitch-mod-opus freeswitch-mod-png freeswitch-mod-rtc freeswitch-mod-say-en freeswitch-mod-shout
  freeswitch-mod-sndfile freeswitch-mod-sofia freeswitch-mod-spandsp freeswitch-mod-tone-stream freeswitch-mod-valet-parking freeswitch-mod-verto
  freeswitch-mod-voicemail freeswitch-mod-xml-curl freeswitch-mod-xml-rpc libfreeswitch1 libldns3 libltdl7 libodbc1 libopencore-amrnb0
  libsofia-sip-ua0 libspandsp3 libtpl0 unifi-talk-freeswitch-conf
Use 'apt autoremove' to remove them.
The following NEW packages will be installed:
  dialog
0 upgraded, 1 newly installed, 0 to remove and 21 not upgraded.
Need to get 276 kB of archives.
After this operation, 1225 kB of additional disk space will be used.
Get:1 http://deb.debian.org/debian bullseye/main arm64 dialog arm64 1.3-20201126-1 [276 kB]
Fetched 276 kB in 0s (6527 kB/s)
debconf: unable to initialize frontend: Dialog
debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 78, <> line 1.)
debconf: falling back to frontend: Readline
Selecting previously unselected package dialog.
(Reading database ... 112224 files and directories currently installed.)
Preparing to unpack .../dialog_1.3-20201126-1_arm64.deb ...
Unpacking dialog (1.3-20201126-1) ...
Setting up dialog (1.3-20201126-1) ...
Reading package lists...
Building dependency tree...
Reading state information...
The following packages were automatically installed and are no longer required:
  freeswitch freeswitch-mod-amr freeswitch-mod-b64 freeswitch-mod-commands freeswitch-mod-conference freeswitch-mod-console freeswitch-mod-db
  freeswitch-mod-dialplan-asterisk freeswitch-mod-dialplan-xml freeswitch-mod-dptools freeswitch-mod-enum freeswitch-mod-esf
  freeswitch-mod-event-socket freeswitch-mod-expr freeswitch-mod-fifo freeswitch-mod-fsv freeswitch-mod-g723-1 freeswitch-mod-g729
  freeswitch-mod-hash freeswitch-mod-httapi freeswitch-mod-local-stream freeswitch-mod-logfile freeswitch-mod-loopback freeswitch-mod-lua
  freeswitch-mod-native-file freeswitch-mod-opus freeswitch-mod-png freeswitch-mod-rtc freeswitch-mod-say-en freeswitch-mod-shout
  freeswitch-mod-sndfile freeswitch-mod-sofia freeswitch-mod-spandsp freeswitch-mod-tone-stream freeswitch-mod-valet-parking freeswitch-mod-verto
  freeswitch-mod-voicemail freeswitch-mod-xml-curl freeswitch-mod-xml-rpc libfreeswitch1 libldns3 libltdl7 libodbc1 libopencore-amrnb0
  libsofia-sip-ua0 libspandsp3 libtpl0 unifi-talk-freeswitch-conf
Use 'apt autoremove' to remove them.
The following NEW packages will be installed:
  igmpproxy udm-iptv
0 upgraded, 2 newly installed, 0 to remove and 21 not upgraded.
Need to get 38.3 kB of archives.
After this operation, 167 kB of additional disk space will be used.
Get:1 /tmp/tmp.ghoVpSfMx0/udm-iptv.deb udm-iptv all 3.0.2 [16.3 kB]
Get:2 http://deb.debian.org/debian bullseye/main arm64 igmpproxy arm64 0.3-1 [38.3 kB]
Fetched 38.3 kB in 0s (870 kB/s)
Preconfiguring packages ...
/tmp/udm-iptv.config.ARPZMN: 142: .: cannot open /usr/lib/udm-iptv/profiles/kpn.profile: No such file
udm-iptv failed to preconfigure, with exit status 2
Selecting previously unselected package igmpproxy.
(Reading database ... 112387 files and directories currently installed.)
Preparing to unpack .../igmpproxy_0.3-1_arm64.deb ...
Unpacking igmpproxy (0.3-1) ...
Selecting previously unselected package udm-iptv.
Preparing to unpack .../tmp.ghoVpSfMx0/udm-iptv.deb ...
Unpacking udm-iptv (3.0.2) ...
Setting up igmpproxy (0.3-1) ...
Setting up udm-iptv (3.0.2) ...
**W: Repository is broken: udm-iptv:arm64 (= 3.0.2) has no Size information**
Installation successful... You can find your configuration at /etc/udm-iptv.conf.

It does not automaticly start the configuration proces so I give the command **udm-iptv reconfigure** en fill in the required information.

The service does seem to start but my IPTV-box gives me the 563 error.

When i check the log-files there seems to be a problem.

Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth9
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br2
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
=== Service Logs ===
Aug 02 14:00:27 Dream-Machine-Special-Edition udm-iptvd[100429]: udhcpc: received SIGTERM
Aug 02 14:00:27 Dream-Machine-Special-Edition udm-iptvd[100429]: udhcpc: entering released state
Aug 02 14:00:28 Dream-Machine-Special-Edition systemd[1]: udm-iptv.service: Succeeded.
Aug 02 14:00:28 Dream-Machine-Special-Edition systemd[1]: Stopped IPTV support for the UniFi Dream Machine.
Aug 02 14:00:28 Dream-Machine-Special-Edition systemd[1]: Started IPTV support for the UniFi Dream Machine.
Aug 02 14:00:28 Dream-Machine-Special-Edition udm-iptvd[100607]: Device iptv already exists.. Deleting device
Aug 02 14:00:28 Dream-Machine-Special-Edition udm-iptvd[100607]: Obtaining IP address for VLAN interface
Aug 02 14:00:28 Dream-Machine-Special-Edition udm-iptvd[100618]: udhcpc: started, v1.30.1
Aug 02 14:00:28 Dream-Machine-Special-Edition udm-iptvd[100618]: udhcpc: sending discover
Aug 02 14:00:31 Dream-Machine-Special-Edition udm-iptvd[100618]: udhcpc: sending discover
Aug 02 14:00:37 Dream-Machine-Special-Edition udm-iptvd[100618]: udhcpc: no lease, forking to background
Aug 02 14:00:37 Dream-Machine-Special-Edition udm-iptvd[100607]: NATing IPTV network ranges (if necessary)
Aug 02 14:00:37 Dream-Machine-Special-Edition udm-iptvd[100607]: Creating static routes (if necessary)
Aug 02 14:00:37 Dream-Machine-Special-Edition udm-iptvd[100607]: Setting up IGMP Proxy
Aug 02 14:00:37 Dream-Machine-Special-Edition udm-iptvd[100607]: Using igmpproxy...
Aug 02 14:00:37 Dream-Machine-Special-Edition udm-iptvd[100607]: Starting IGMP Proxy
Aug 02 14:00:37 Dream-Machine-Special-Edition udm-iptvd[100607]: adding VIF, Ix 0 Fl 0x0 IP 0x0103a8c0 br2, Threshold: 1, Ratelimit: 0
Aug 02 14:00:37 Dream-Machine-Special-Edition udm-iptvd[100607]: There must be at least 1 Vif as upstream.
Aug 02 14:00:37 Dream-Machine-Special-Edition systemd[1]: udm-iptv.service: Main process exited, code=exited, status=255/EXCEPTION
Aug 02 14:00:37 Dream-Machine-Special-Edition systemd[1]: udm-iptv.service: Failed with result 'exit-code'.
Aug 02 14:00:42 Dream-Machine-Special-Edition systemd[1]: udm-iptv.service: Scheduled restart job, restart counter is at 1.
Aug 02 14:00:42 Dream-Machine-Special-Edition systemd[1]: Stopped IPTV support for the UniFi Dream Machine.
Aug 02 14:00:42 Dream-Machine-Special-Edition systemd[1]: Started IPTV support for the UniFi Dream Machine.
Aug 02 14:00:42 Dream-Machine-Special-Edition udm-iptvd[101083]: Device iptv already exists.. Deleting device
Aug 02 14:00:42 Dream-Machine-Special-Edition udm-iptvd[101083]: Obtaining IP address for VLAN interface
Aug 02 14:00:42 Dream-Machine-Special-Edition udm-iptvd[101097]: udhcpc: started, v1.30.1
Aug 02 14:00:42 Dream-Machine-Special-Edition udm-iptvd[101097]: udhcpc: sending discover
Aug 02 14:00:45 Dream-Machine-Special-Edition udm-iptvd[101097]: udhcpc: sending discover

It looks like it keeps getting an error and restarts, this proces repeats itself over and over again.

My internet connection comes in at port 9 (GBE) and the IPTV device is on VLAN 2.

Before this probleme the script had been running without a problem for the last couple of months.

Any ideas what I'm doing wrong?

**Comment by sulzer** [comment]
Hi @Grasio 

I seem to be experiencing the same on my UDM SE since I've upgraded to firmware 3.1.15:
https://github.com/fabianishere/udm-iptv/discussions/269

When I uninstall and reinstall udm-iptv I get this error as well:
Repository is broken: udm-iptv:arm64 (= 3.0.2) has no Size information

Although it says "Installation successful" a line further down.

@fabianishere Do you have any idea what could cause this error?

Thank you so much.

**Comment by fabianishere** [comment]
This is a warning that you can ignore for now. Looking at the error logs, it looks to have trouble obtaining an IP address on WAN VLAN 4. Are you sure it is configured correctly?
